portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java

```diff
@@ -6335,7 +6335,7 @@ protected void notifyUser(User user, ServiceContext serviceContext)
 			passwordResetURL = StringBundler.concat(
 				serviceContext.getPortalURL(), serviceContext.getPathMain(),
 				updatePasswordURL, "ticketId=", ticket.getTicketId(),
-				"&ticketKey=", ticket.getKey());
+				"&ticketKey=", ticket.getKey(), "&languageId=", user.getLanguageId());
```

Comment:
Can we move it as first parameter? And are we sure the user.getLanguageId() always returns something or we need to check it and, if empty, avoid adding the parameter?

Reply:
@smottal 

Thanks for the comment.

1- This parameter is invisible to the user, it does not appear in the URL

2- Great observation, I did some tests and also confirmed that no user can have an empty LanguageId, every user has some LanguageId

Reply:
@henriquelimadotcom 
1- Yes, the moving is not for the user but for our code maintainability
2- Ok

Reply:
Done